core/src/Kokkos_Assert.hpp

```diff
@@ -20,6 +20,18 @@
 #include <Kokkos_Macros.hpp>
 #include <Kokkos_Abort.hpp>
 
+#define KOKKOS_EXPECTS_CRITICAL(...)                                           \
```

Comment:
```suggestion
#define KOKKOS_IMPL_EXPECTS_CRITICAL(...)                                           \
```

Reply:
Why was it marked as resolved?  It clearly was not.  Did you forget to push?

Reply:
I've probably clicked the wrong button. Now it is commited.